README.md

```diff
@@ -68,6 +68,7 @@ Thanks for checking us out! If you're new here, here are some things you should
  - Ensure your build passes (`rubocop -a` is often necessary) and addresses the issue requirements
  - Check the [Contributing Guidelines](#-contributing-guidelines) for getting started
  - This project relies entirely on volunteers, so please be patient with communication
+  - Visit the [wiki](https://github.com/rubyforgood/human-essentials/wiki) for more information
```

Comment:
The wiki is a bit weak, not sure we wan't to add it

Reply:
Noted. It'd be great to update the wiki in the future and link it here. I think the wiki would be a great place for the deployment process section as it's not relevant to most contributors (I think, could be wrong).